pkg/sql/parser/sql.y

```diff
@@ -5420,11 +5420,21 @@ backup_kms:
 
 // %Help: SHOW TENANT - display tenant information
 // %Category: Misc
-// %Text: SHOW TENANT
+// %Text: SHOW TENANT <name> WITH REPLICATION STATUS
 show_tenant_stmt:
   SHOW TENANT name
   {
-   $$.val = &tree.ShowTenant{Name: tree.Name($3)}
+   $$.val = &tree.ShowTenant{
+     Name: tree.Name($3),
```

Comment:
Given our discussion last week about moving all tenant names to actually be expressions, should we do that here instead of adding more fields to change in the future? I was going to use `d_expr` in `ALTER TENANT`, there is an example of this in `ALTER TENANT CLUSTER SETTINGS`.

Reply:
Done.